ci/Dockerfile.fedora

```diff
@@ -43,13 +43,21 @@ ARG NCPU=2
 ARG CXX=g++
 ARG CC=gcc
 ARG BUILD_TYPE=Debug
+ARG CHECK_STYLE=""
 
 # We assume that this is running on a (clean-ish) checkout of
 # google-cloud-cpp, including submodules, and copy the files to a
 # directory where we will build.
+WORKDIR /var/tmp/build
 COPY . /var/tmp/build/gccpp
+RUN test "${CHECK_STYLE}" != "yes" || cp -r gccpp gccpp-orig
 
 WORKDIR /var/tmp/build/gccpp/build-output
 RUN cmake -DCMAKE_BUILD_TYPE=${BUILD_TYPE} ..
 RUN make -j ${NCPU} all
 RUN make -j ${NCPU} test || ( cat Testing/Temporary/LastTest.log; exit 1 )
+
+RUN test "${CHECK_STYLE}" != "yes" || make clang-format
```

Comment:
Is this the same as `RUN test "${CHECK_STYLE}" == "yes" && make clang-format` ?

If so, the one without negations is easier to read. Thoughts?

Reply:
You want the exit status to be 0 when CHECK_STYLE != "yes".  So the expression is correct.
We can do an inline `if [ "${CHECK_STYLE}" == "yes" ]; then ...; fi` if you prefer.

Reply:
That makes sense, no need to make the change.